.circleci/config.yml

```diff
@@ -203,6 +203,7 @@ jobs:
       - run:
           command: |
             cd mattermost-server
+            go vet ./...
```

Comment:
With this step, we improve the error message in the CI pipeline.
However, wouldn't you include this in the command `make golangci-lint`?

I usually run the command locally and actually, I have a pre-push hook that prevents me from pushing code that doesn't pass the lint.

Reply:
I think `make golangci-lint` should only run golangci. This is mainly to make the error message nicer. Golangci also runs vet by itself. So this isn't adding any extra checks. It's just to make compilation errors appear better.

Reply:
The ideal solution would be golangci-lint running govet before typecheck. However, it doesn't seem to be possible to configure it that way. 

It's true that my suggestion is a workaround that makes govet run twice always.

Reply:
Yep, for sure. We are just covering for the tool's shortcomings :)